.gitignore

```diff
@@ -15,6 +15,7 @@ eshell/lastdir
 my-org/
 org-files/
 semanticdb/
+srecode-map.el
```

Comment:
If it is not related to fasd it can be removed from the PR.
Before adding it to the ignore list we can try to add it to `.cache` directory.